samples/boards/nrf52/power_mgr/README.rst

```diff
@@ -0,0 +1,58 @@
+.. _power-mgr-sample:
```

Comment:
/samples/boards/quark_se_c1000/power_mgr/README.rst has this same label (labels must be unique across all documents).  Maybe call this "nrf52-power-mgr-sample"?